daemon/src/payout_curve.rs

```diff
@@ -1,13 +1,30 @@
 use crate::model::{Leverage, Usd};
 use anyhow::Result;
 use bdk::bitcoin;
+use cfd_protocol::interval::MAX_PRICE_DEC;
 use cfd_protocol::Payout;
 
 pub fn calculate(
-    _price: Usd,
+    price: Usd,
     _quantity: Usd,
-    _maker_payin: bitcoin::Amount,
-    (_taker_payin, _leverage): (bitcoin::Amount, Leverage),
+    maker_payin: bitcoin::Amount,
+    (taker_payin, _leverage): (bitcoin::Amount, Leverage),
 ) -> Result<Vec<Payout>> {
-    Ok(vec![])
+    let dollars = price.try_into_u64()?;
+    let payouts = vec![
+        Payout::new(
+            0..=(dollars - 10),
+            maker_payin + taker_payin,
+            bitcoin::Amount::ZERO,
+        )?,
+        Payout::new((dollars - 10)..=(dollars + 10), maker_payin, taker_payin)?,
```

Comment:
:wrench: It may not matter for the dummy payouts, but they ranges are inclusive. Here there would be slight overlaps between some CETs, which is bad.

Reply:
A right, didn't realize this.